File: examples/rss/feed/node/readAFeed.js
const { RssService } = require("m3o/rss");

// Get an RSS feed by name. If no name is given, all feeds are returned. Default limit is 25 entries.
async function readAfeed() {
  let rssService = new RssService(process.env.MICRO_API_TOKEN);
  let rsp = await rssService.feed({
    name: "bbc",
  });
  console.log(rsp);
}

readAfeed();

File: examples/rss/feed/go/readAFeed.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/rss"
	"os"
)

// Get an RSS feed by name. If no name is given, all feeds are returned. Default limit is 25 entries.
func ReadAfeed() {
	rssService := rss.NewRssService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := rssService.Feed(&rss.FeedRequest{
		Name: "bbc",
	})
	fmt.Println(rsp, err)
}
